SuiteCRM/SuiteCRM
# add option to limit number of records on each workflow run

This feature allows to limit the number of records (beans) on each workflow run.
![image](https://user-images.githubusercontent.com/1040582/100750229-6387a880-33ee-11eb-8f81-8d65e8254f3c.png)

And this is how it looks like with [workflow period of time feature](https://github.com/salesagility/SuiteCRM/pull/8831):
![image](https://user-images.githubusercontent.com/1040582/100750805-38ea1f80-33ef-11eb-8648-12ae28810aba.png)

## Description





Today, when workflow is run on crontab, it will take all beans populated from the workflow conditions. On some scenarios you need to limit the number of beans on each run to make other workflows run more iteratively and avoid all other processes to be stuck for a long time.

## Motivation and Context

Control on the crontab workflow run.

## How To Test This

* Regression test: put 0 on the limit beans field and see that all beans are taken.
* Take workflow with X beans on one run, and put number of X/3. Verify the workflow takes all beans ub 3 runs.

## Types of changes


### Final checklist
